doc/rules/control_structure/control_structure_continuation.rst

```diff
@@ -0,0 +1,56 @@
+=======================================
+Rule ``control_structure_continuation``
+=======================================
+
+Control structure continuation keyword must be on the configured line.
+
+Configuration
+-------------
+
+``keyword_position``
```

Comment:
Wouldn't simply `position` be enough? Maybe the fixer then could be renamed to `control_structure_continuation_position`?

Reply:
Done.